source/subs_functions/subs_functions.bas

```diff
@@ -3867,3 +3867,31 @@ id.arg = MKL$(UINTEGER64TYPE - ISPOINTER) + MKL$(LONGTYPE - ISPOINTER)
 id.ret = UINTEGER64TYPE - ISPOINTER
 id.hr_syntax = "_TOGGLEBIT(numericalVariable, numericalValue)"
 regid
+
+clearid
+id.n = qb64prefix$ + "WindowWidth"
+id.subfunc = 1
+id.callname = "func__scaledwidth"
+id.args = 0
+id.ret = UINTEGER64TYPE - ISPOINTER
+id.hr_syntax = "_WINDOWWIDTH"
+regid
+
+clearid
+id.n = qb64prefix$ + "WindowHeight"
+id.subfunc = 1
+id.callname = "func__scaledheight"
+id.args = 0
+id.ret = LONGTYPE - ISPOINTER
+id.hr_syntax = "_WINDOWHEIGHT"
+regid
+
+clearid
+id.n = qb64prefix$ + "WindowScale"
+id.subfunc = 2
+id.callname = "sub__windowscale"
+id.args = 1
+id.arg = MKL$(LONGTYPE - ISPOINTER)
+id.hr_syntax = "_WINDOWSCALE(amount)"

```

Comment:
Since this is a command, should it be `_WINDOWSCALE amount`? I'm basing that off of the other commands in this file :D

Reply:
I'll double check the syntax here.  You're probably right.  ;)